src/platform/EFR32/BUILD.gn

```diff
@@ -43,6 +44,8 @@ static_library("EFR32") {
     "DiagnosticDataProviderImpl.h",
     "EFR32Config.cpp",
     "EFR32Config.h",
+    "EFR32DeviceAttestationCreds.cpp",
+    "EFR32DeviceAttestationCreds.h",
```

Comment:
All of this should be under `examples/platform/efr32/BUILD.gn`, so that this way you have the right dependency chain. platform/ cannot cleanly depend on credentials/

Reply:
Moved under `examples/platform/efr32/`